modules/core/server/controllers/core.server.controller.js

```diff
@@ -14,6 +14,7 @@ exports.renderIndex = function (req, res) {
       displayName: validator.escape(req.user.displayName),
       provider: validator.escape(req.user.provider),
       username: validator.escape(req.user.username),
+      _id: req.user._id,
```

Comment:
This would indeed fix the issue. However, we purposely removed the `_id` field from the client-side User object for reasons of security. I don't see an easy way to handle this on the client-side.

One thing for sure is that we need to have a check on the server-side, to prevent a user from deleting their own account. This can easily be done [here](https://github.com/meanjs/mean/blob/master/modules/users/server/controllers/admin.server.controller.js#L46) by adding something like:
```javascript
if (user._id === req.user._id) {
  return res.status(400).send({
      message: 'You cannot delete yourself!'
    });
}
```

Additionally, we can do something similar to [this](https://github.com/meanjs/mean/blob/master/modules/articles/server/controllers/articles.server.controller.js#L36-L38). Which we would then have a field attached to the User that is being "edited" that specifies whether or not the current active User is the same as the one being edited. This field could be named something like `isCurrentUserSelf` (or something more clever), and this field would NOT be persisted to the database.